client/src/components/Toolshed/InstalledList/Monitor.vue

```diff
@@ -59,7 +59,7 @@ export default {
     },
     created() {
         this.root = getAppRoot();
```

Comment:
Similar to the case above, this can be removed together with the import of `getAppRoot`.